modules/calib3d/perf/perf_stereosgbm.cpp

```diff
@@ -78,6 +78,33 @@ PERF_TEST_P( TestStereoCorresp, DISABLED_TooLongInDebug_SGBM, Combine(Values(Siz
     SANITY_CHECK(dst, .01, ERROR_RELATIVE);
 }
 
+typedef tuple<Size, int> BMParams;
+typedef TestBaseWithParam<BMParams> TestStereoCorrespBM;
+
+PERF_TEST_P(TestStereoCorrespBM, BM, Combine(Values(Size(1280, 720), Size(640, 480)), Values(256, 128)))
+{
+    RNG rng(0);
```

Comment:
This is not needed in general:
https://github.com/opencv/opencv/blob/4.1.1/modules/ts/src/ts.cpp#L758-L763

Reply:
Default seed is non zero, so this change will affect sanity result for some test cases. I could remove it for BM test only and regenerate new sanity data, but IMO it would be better to retain similar tests uniform. The other option(the one I actually prefer) is to move zero RNG initialization to the `MakeArtificialExample` function that produce input to make things more consistent. The third option is to remove RNG initialization completely and regenerate both BM and SGBM sanity data.